docs/_posts/2018-07-04-chatopera-values-vision.md

```diff
@@ -135,7 +135,7 @@ Chatopera产品线
 
 对我们来说，做用户喜爱的产品就意味着，用户能从我们的产品中感受到热情，效率和尽善尽美，选择我们就是选择成功。
 
-![Chatopera创始团队](/assets/2018/07-chatopera-values-vision/founding-team.jpg)
+![Chatopera创始团队](/assets/2018/07-chatopera-values-vision/founding-team.png)
```

Comment:
It seems that you replace the JPG file with PNG for purpose.

Cloud you explain the reason behind that? Because I'm very courious about it, thank you very much.